gax/package.json

```diff
@@ -2,13 +2,42 @@
   "name": "google-gax",
   "version": "4.5.0",
   "description": "Google API Extensions",
-  "main": "build/src/index.js",
-  "types": "build/src/index.d.ts",
+  "main": "./build/cjs/src/index.cjs",
+  "type": "module",
+  "types": "./build/cjs/src/index.d.ts",
```

Comment:
Webpack 4- and earlier versions of eslint will fail here.

Additionally, in the storage example the `main` ends with `.js` rather than `.cjs`.

Reply:
I think of it not whether we claim to support it or not, but rather are we following guidance and standards appropriately. If not, we can expect some customers to run into friction.

Reply:
Hm, isn't storage still using a cjs file (seems like the js file is in the cjs module?) Also we are running tests using webpack 4 for the browser and they are WAI!

Reply:
Regardless, I don't feel strongly about this since we're testing gax in our system tests, and I think the exports field have precedence over main anyways. Running tests to make sure everything passes.

Reply:
Update, seems like this actually breaks browser tests: https://btx.cloud.google.com/invocations/37842334-df7a-4bd8-8aef-a97b319b0cd7/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fgax-nodejs%2Fnode18%2Fbrowser-test/log

Reply:
FWIW, this was my attempt when adding back in a `browser` field to default to cjs and then use esm as the entrypoint: https://btx.cloud.google.com/invocations/feb590d8-03ab-40b6-bf59-fcd445db5f47/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fgax-nodejs%2Fnode18%2Fbrowser-test/log